app/models/user.rb

```diff
@@ -10,4 +10,8 @@ class User < ApplicationRecord
   # Roles
   CUSTOMER = 0
   ADMIN = 1
+
+  def admin?
+    role == ADMIN
```

Comment:
Have the "ADMIN" in some constant and try to compare with that constant, instead of comparing hard coded value.

Reply:
"ADMIN" is already set as a constant with a value of 1. This constant is then used in the admin? method for comparison.

Reply:
Yeah Cool